libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/EntitySection.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    /// <summary>SectionEntity class.</summary>
+    public partial class SectionEntity
+    {
+        // TODO: pass this constant to a helper class.
+        private readonly char[] _invalidCharsInIntentOrEntityName = { '<', '>', '*', '%', '&', ':', '\\', '$' };
+
+        public SectionEntity(LUFileParser.EntitySectionContext parseTree)
+        {
+            SectionType = SectionType.EntitySection;
+            Errors = new List<Error>();
```

Comment:
Brainstorm / Question - instead of defining `Error` and having a `List<Error>` I wonder if it would be useful to just use built-in exceptions and wrap in an aggregate exception. Not sure if that is the way to go, just brainstorming.